sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/ShowCreateTableExec.scala

```diff
@@ -98,11 +126,11 @@ case class ShowCreateTableExec(
       builder: StringBuilder,
       tableOptions: Map[String, String]): Unit = {
 
-
     val showProps = table.properties.asScala
       .filterKeys(key => !CatalogV2Util.TABLE_RESERVED_PROPERTIES.contains(key)
         && !key.startsWith(TableCatalog.OPTION_PREFIX)
-        && !tableOptions.contains(key))
+        && !tableOptions.contains(key)
+        && !key.equals(TableCatalog.PROP_EXTERNAL))
```

Comment:
isn't it converted by `TABLE_RESERVED_PROPERTIES`?

Reply:
As `PROP_EXTERNA` is not include by `TABLE_RESERVED_PROPERTIES`. do you think it should be included by `TABLE_RESERVED_PROPERTIES` ?

Reply:
yea it should, it's indeed a reserved property

Reply:
done.